src/features/trigger/CronTrigger.tsx

```diff
@@ -50,25 +65,33 @@ const CronTrigger: React.FC<CronTriggerProps> = ({
       displayInterval = ''
   }
 
+  const onEdit = () => {
+    if(!isEditable){
+      setIsEditable(true)
+    }
+  }
+
   return (
-    <Block>
-      <div className='flex'>
-        <div className='flex-grow'>
-          <div id='cron_trigger_schedule' className='text-sm font-semibold pb-1 text-black'>Schedule</div>
-          <div className='text-qrigray-400 text-xs flex items-center'>
+    <div onClick={onEdit} className='w-full cursor-pointer'>
+      <div className='flex w-full items-center'>
+        <div className='flex-grow flex items-center'>
+          <div id='cron_trigger_schedule' className='text-sm font-semibold mr-2.5 text-black'>Schedule</div>
+          {!isEditable &&<div className='text-qrigray-400 text-xs flex items-center'>
             <Icon icon='calendar' size='sm' className='mr-1 pt-1' />
             <div id='cron_trigger_interval_text'>{displayInterval} {displayStartTime}</div>
-          </div>
-        </div>
-        <div className='flex ml-2'>
-          <Link
-            className='text-qrigray-400 text-xs self-end mb-0.5'
-            colorClassName='text-qrigray-400 hover:text-qrigray-600'
-            onClick={handleEditClick}
-          >Edit</Link>
+          </div>}
         </div>
       </div>
-    </Block>
+      {isEditable &&
+      <div className='w-full'>
+        <div className='text-qrigray-700 text-xs'>Repeat</div>
+        <CronTriggerEditor
+          trigger={triggers[0]}
```

Comment:
getting a typescript error here as well. Once we cast `trigger[0]` as a `CronTrigger`, we can use that same variable to pass as the `trigger` into `CronTriggerEditor`.